code/hispania/modules/mob/language/station.dm

```diff
@@ -0,0 +1,19 @@
+/datum/language/kidan
+	name = LANGUAGE_KIDAN
+	desc = "The noise made by rubbing its antennae together is actually a complex form of communication for Kidan."
```

Comment:
more race thing even a new language. i think you merged either in the wrong codebase. or the wrong branch